Survey reps

fatiabdulahi opened the pull request:
it's the survey project Iwork html and css

fatiabdulahi committed:
Merge pull request #1 from fatiabdulahi/feature

 i createat my sarvey form

fatiabdulahi committed:
Merge pull request #2 from fatiabdulahi/feature

first commit

fatiabdulahi committed:
Update README.md

# Project Title 🚀 Survey Form Project

## Project Description 📝

> This is a survey form for getting information about girls education.
which the main purpose of this survey form is : How many people agreed that girls go to school?

## Technologies Used 🛠️

- HTML
- CSS

## Author 👩‍💻

Fatima Abdullahi

- Email: [ fatima333abdullahi@gmail.com)

fatiabdulahi commented:
@Saqibi4213 dear mehri please read again and shear your openean

fatiabdulahi commented:
@ahzamir hello dear Sir please read my project and shear your opinean

Saqibi4213 commented:
still your readme.md file is not professional as the required in the instruction you should follow all instruction in folder of project.readme file thier is clear what you should have you delete the other instruction from readme this is not done yet please try again @fatiabdulahi  and also you should deploy your project and do it with all project

fatiabdulahi committed:
Update README.md

# Survey form  🚀

## Project Description 📝

> This is a survey form for getting information about girls education.
which the main purpose of this survey form is : How many people agreed that girls go to school?



## Demo 📸

![Demo](./img/survey-form.png)

## Technologies Used 🛠️

- HTML
- CSS

## Installation 💻

Folow these steps to set up the project on your local machine 
Ensure you have the following software installed on your machine

```bash
#git clone
```

```bash
#git@github.com:fatiabdulahi/Fati-sarvey.git
```

## Usage 🎯

you can access to my project through git clone 
-git clone the HTTP or My SSH code link from the repository you wat to clone 

``` 
#git clone
```

## Features ⭐

1: General Information
*Name:*
*Email:*
*Age:*
*Gender:*
2: Kitchen Use and Preferences
3: Food Taste Preferences
4: Suggestions and Feedback
## Author 👩‍💻

## Author 👩‍💻

Fatima Abdullahi

- Email: [ fatima333abdullahi@gmail.com)
 -Website:  https://saqibi4213.github.io/survey-form/

## Contributing 🤝

We welcome contributions from the community! Please follow the guidelines below to contribute to our project.
<br />
1: Fork the Repository: Click the "Fork" button at the top right of the repository page to create a copy of the repository on your GitHub account.
<br />
2: Clone Your Fork: Clone your forked repository to your local machine using the following command:git clone https://github.com/yourusername/yourprojectname.git
<br />
3: Create a Branch: Create a new branch for your work to ensure your changes are isolated from the main branch:git checkout -b feature-or-bugfix-description
<br />
4: Make Your Changes: Implement your changes, whether fixing a bug or adding a new feature. Be sure to follow the project's coding standards and guidelines.
<br />
5: Commit Your Changes: Commit your changes with a descriptive commit message:git add .
git commit -m "Description of the changes"
<br />
6: Push to Your Fork: Push your changes to your forked repository:git push origin feature-or-bugfix-description
<br />
7: Submit a Pull Request: Go to the original repository and submit a pull request from your forked repository. Provide a clear and concise description of your changes and why they should be merged..

fatiabdulahi committed:
Update README.md

fatiabdulahi commented:
@Saqibi4213 Hi dear mehri I changed read me pleas read again and shear your openian

Saqibi4213 commented:
still your readme is not complete and not deployed your 
@fatiabdulahi

Saqibi4213 commented:
still your readme is not complete and not deployed your

fatiabdulahi committed:
Update README.md

fatiabdulahi commented:
@Saqibi4213 dear Mehre i edit my read me pleas revewo and shear your sujjestion.

Saqibi4213 commented:
- still need to make your readme professional 
- 
![Screenshot (33)](https://github.com/fatiabdulahi/Fati-sarvey/assets/168208262/22d2a30e-5d5e-49a5-bb5b-8cbcb39de1ab)

- and you add your all other description of readme in ``` ()```
- remove ``` from feature and as you can see their is still error in readme you should fix it 
![Screenshot (34)](https://github.com/fatiabdulahi/Fati-sarvey/assets/168208262/405d8629-d6d9-43e6-abb4-775f1bc7075c)
@fatiabdulahi

ahzamir commented:
@fatiabdulahi 

- You created a conflict cause you changed your readme file on the development branch so solve that and then finish all the required changes your partner requested.
